signalfx-org-metrics/metrics.yaml

```diff
@@ -1457,6 +1457,164 @@ sf.org.limit.metricTimeSeriesCreatedPerMinute:
   metric_type: gauge
   title: sf.org.limit.metricTimeSeriesCreatedPerMinute
 
-
+sf.org.log.numMessagesReceived:
+  brief: The total number of log messages Splunk Log Observer accepts after filtering and throttling.
+  description: |
+   The total number of log messages Splunk Log Observer accepts after filtering and throttling.
+ 
+       * Dimension(s): `orgId`
+       * Data resolution: 10 second
```

Comment:
not sure where this Data resolution is coming from

Reply:
<img width="933" alt="MetricDataRes" src="https://user-images.githubusercontent.com/57269043/129403584-4cc18a47-ca4a-4c13-a931-b8529c07dd06.png">

Reply:
@jgheewala I looked up the metric in Metric Finder and found the data resolution next to the metric name after clicking into the metric. This was the guidance I received. Is it correct? That's where I pulled all of the Data Resolution values from. 
cc: @mstumpfx @jhasegawa-splunk

Reply:
ohh a customer can change that resolution to `30s` and that resolution can change. But i guess what you are looking at it the min resolution that we can get ?

Reply:
you can look at resolution that's being advertised by splunk apm product as that will be exactly the same.